src/renderer/pages/Operations/model/reject-model.ts

```diff
@@ -21,7 +20,7 @@ type GetMultisigType = {
   tx: MultisigTransactionDS | FlexibleMultisigTransactionDS;
 };
 
-const flow = createGate<{ chain: Chain; signer: AnyAccount }>();
+const flow = createGate<{ chain: Chain | null; signer: AnyAccount | null }>();
```

Comment:
In this case use `defaultState`
```suggestion
const flow = createGate<{ chain: Chain | null; signer: AnyAccount | null }>({ defaultState: { chain: null, signer: null } });
```